src/mbgl/shader/circle_vertex.cpp

```diff
@@ -0,0 +1,13 @@
+#include <mbgl/shader/circle_vertex.hpp>
+#include <mbgl/shader/shader.hpp>
+#include <mbgl/gl/gl.hpp>
+
+namespace mbgl {
+
+void CircleVertex::bind(const int8_t* offset) {
+    static_assert(sizeof(CircleVertex) == 4, "expected CircleVertex size");
+
+    MBGL_BIND_VERTEX_ATTRIBUTE(CircleVertex, a_pos, offset);
```

Comment:
This is an incorrect place for using `a_pos`; those names/attributes are shader-specific and not Vertex type specific.